packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -170,24 +170,13 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
             }
 
             if (isTimestamp) {
-                const parsedTimestampToUtc = rule.values
-                    ? dayjs(
-                          parseTimestamp(
-                              formatTimestamp(
-                                  rule.values[0],
-                                  TimeFrames.MILLISECOND,
-                              ),
-                              TimeFrames.MILLISECOND,
-                          ),
-                      ).utc()
-                    : null;
+                // For display only
+                const displayFormat = 'YYYY-MM-DD HH:mm:ss (Z)';
```

Comment:
just checking that this change hasn't affected this one: https://github.com/lightdash/lightdash/issues/8942 with the PR that fixed it here: https://github.com/lightdash/lightdash/pull/9303

Reply:
After chatting with @magnew the approach done here: https://github.com/lightdash/lightdash/pull/9303 is not recommended and didn't fully solve the issue. Thanks for demo'ing the problem!